modules/install/pages/extensions.adoc

```diff
@@ -1,24 +1,316 @@
-= Extensions, Subsystems, and Profiles
+= Interpreting Extensions, Subsystems, and Profiles
 
-Brief blurb about the architecture of JBoss EAP (Do not repeat what will be already be covered in the Tech Overview course 1 in the path)
+* JBoss EAP 8.0 provides implementations of the Jakarta EE 10 APIs and Jakarta EE 10 https://jakarta.ee/specifications/webprofile/10/[Web Profile], https://jakarta.ee/specifications/coreprofile/10/[Core Profile], and the Full https://jakarta.ee/specifications/platform/10/[Platform] standards.
+* JBoss EAP 7.4, supported Jakarta EE 8. However, Jakarta EE 10 has undergone several changes, the most significant being the modification of the package namespace for Jakarta EE APIs from javax to jakarta.
+* See the complete list of specifications in the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/8.0/html-single/release_notes_for_red_hat_jboss_enterprise_application_platform_8.0/index#jakarta-ee-10-support_assembly-release-notes[release notes].
+
+== Modules in JBoss EAP
+
+* A module is a bundle composed of a library developed in Java and an XML configuration file that provides Jakarta Enterprise Edition (Jakarta EE) compliant functionalities.
+* The modules are located at the `JBOSS_HOME/modules/system/layers/base/` directory.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ ls -l /opt/EAP-8.0.0/modules/system/layers/base/
+total 16
+drwxr-xr-x.  3 jboss jboss   17 Apr  9 04:13 asm
+drwxr-xr-x. 15 jboss jboss 4096 Apr  9 04:13 com
+drwxr-xr-x.  3 jboss jboss   20 Apr  9 04:13 gnu
+drwxr-xr-x.  3 jboss jboss   17 Apr  9 04:13 ibm
+drwxr-xr-x.  3 jboss jboss   19 Apr  9 04:13 internal
+drwxr-xr-x.  6 jboss jboss   68 Apr  9 04:13 io
+drwxr-xr-x. 23 jboss jboss 4096 Apr  9 04:13 jakarta
+drwxr-xr-x.  3 jboss jboss   17 Apr  9 04:13 javaee
+drwxr-xr-x. 29 jboss jboss 4096 Apr  9 04:13 javax
+drwxr-xr-x.  4 jboss jboss   35 Apr  9 04:13 net
+drwxr-xr-x. 29 jboss jboss 4096 Apr  9 04:13 org
+drwxr-xr-x.  4 jboss jboss   34 Apr  9 04:13 sun
+drwxr-xr-x.  3 jboss jboss   17 Apr  9 04:13 wildflyee
+----
 
 == Extensions
 
-== Subsystems
+* Extensions are modules that extend the core capabilities of the server.
```

Comment:
confusing. do you mean modules contain core functionality and extensions extend the feature set. simpler languages needed

Reply:
reframed and simplified